.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -0,0 +1,22 @@
+name: "🐛 Bug Report"
+description: Create a bug report for React Pill.
+body:
+  - type: textarea
+    attributes:
+      label: Describe the Bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Console log output (if relevant)
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```

Comment:
thank you for your contribution.
please add an option to add a screenshot of bug

Reply:
Done